.claude/commands/setup-new-repo.md

```
@@ -55,6 +57,53 @@ ls -la TARGET_DIR 2>/dev/null || echo "Directory will be created"
+6. pnpm-workspace.yaml / lerna.json が存在 → monorepo
+7. *.tf ファイルが存在 → terraform
+8. package.json に bin / exports フィールド → npm-library
+9. package.json のみ → nodejs
+10. いずれも該当しない → unknown（ユーザーに確認）
+```
+
+### 3.5.3 検出結果の確認
+
+自動検出された場合、結果をユーザーに表示して確認：
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)